downloads/swan-lake-release-notes/swan-lake-2201.2.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,759 @@
+
+##### `websocket` package
+
+- Added constraint validation to WebSocket payload binding
+
+##### `graphql` package
+
+- Added the support for deprecation of fields and enum values
+- Added the support for GraphQL interceptors
+
+##### `serdes` package
+
+- Introduced the `serdes` standard library package for serializing and deserializing Ballerina `anydata` subtypes
+- Proto3 is the underlying technology used by this package to achieve serialization and deserialization
+
+##### `os` Package
+- Introduced the `exec()` function to support OS command execution in Ballerina
+
+##### `xmldata` package
+
+- Introduced new APIs such as `fromXml` and `toXml` to perform conversions between `XML` and `map<anydata>`. 
+  The `toRecord` API is being deprecated by introducing this `fromXml` API
+- Introduced a new config named `rootTag` in the `JsonOptions` to configure the name of the XML root element tag
+
+##### `sql` Package
+- Added schema client abstraction to support metadata retrieval from SQL databases. The implementation for the connectors will be added soon
+
+##### `grpc` Package
+- Introduced message-level annotations for the proto descriptor instead of a centralized proto descriptor
+- Introduced packaging support
+- Added stub generation support for nested directories
+
+##### `kafka` Package
+- Added constraint validation support for payload binding
+
+##### `rabbitmq` Package
+- Added constraint validation support for payload binding
+
+##### `nats` Package
+- Added constraint validation support for payload binding
+
+#### Improvements
+
+##### `http` package
+
+- Made HTTP2 the default transport for the `http` package
+- Updated the default response status as `HTTP 201` for POST resources
+
+##### `random` Package
+- Updated the `createDecimal()` function to be cryptographically secure
+
+##### `grpc` Package
+- Added sample client calls with dummy values to generated client files
+- Removed caller client object when generating code in client mode
+- Added a new `grpc:Descriptor` annotation for services as a replacement for the current `grpc:ServiceDescriptor`. Both annotations are supported now to maintain backward compatibility. The `grpc:ServiceDescriptor` will be removed in the future. (Please update the service annotation if stub files are regenerated for the existing gRPC services)
+
+#### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.2.0%22+label%3AType%2FBug).
+
+### Code to Cloud updates
+
+#### Improvements
+The base image was updated to `ballerina/jvm-runtime:1.0` based on Alpine 3.15 with the necessary libraries.
+
+#### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+2201.2.0%22+label%3AType%2FBug).
```

Comment:
Seems it has no bugs. So can't we remove the bug fixes section? @xlight05

Reply:
Theres one, I missed the milestone. Added now.